scripts/prepare_automatic_backups.py

```diff
@@ -80,21 +99,37 @@ def update_cron_dict(cron_dict):
 
 
 def get_cron_dict():
+    """Getting cron file content in dict format.
+
+    Returns:
+        dict(str, str). yaml file in dict format.
+            'description': str. {{ weekly backup.}}
+            'url': str. {{current backup url.}}
+            'schedule': str. {{sheduled time of cron job.}}
+            'target': str. {{cloud storage name.}}
```

Comment:
Ditto. Please remove all the curly braces.

Reply:
resolved it